EndaYohannesQollaGable/EYQG041.xml

```diff
@@ -0,0 +1,144 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="EYQG041" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>The Life-Giving Homily</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+               <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+               <pubPlace>Hamburg</pubPlace>
+               <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                  Forschungsumgebung / Beta maṣāḥǝft</publisher>
+               <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                     This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1008EndaYohannes"/>
+                        <idno>EYQG-041</idno>
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                       <msItem xml:id="ms_i1">                        
+                           <title type="complete" ref="LIT1291Dersan" cert="medium">The Life-Giving Homily</title>
+                         <textLang mainLang="gez"/>
+                           <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ቅዱስ፡ በስመ፡ ወልድ፡ ቅዱስ፡ በስመ፡ መ</hi>ንፈስ፡ ቅዱስ፡ ፩እግዚአብሔር፡ ጸባዖት፡ 
+                               ፍጹመ፡ ዘመልአ፡ ሰማያት፡ ወ<hi rend="rubric">ምድረ፡ ቅድሳተ> ስብሐቲከ፡ አአምን፡ ወእትአ</hi>ምን፡ እሰግድ፡ ለአ<hi rend="rubric">ብ፡</hi>
+                               ወወ<hi rend="rubric">ል</hi>ድ፡ <hi rend="rubric">ወ</hi>መ<hi rend="rubric">ን</hi>ፈ<hi rend="rubric">ስ፡</hi> ቅዱስ፡ ዘይነብር፡ ወትረ፡ በዋህድናሁ።</incipit>
+                          
+                          
+                       </msItem>
+                      
+                                                                     
+                        </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>                          
+                                           <material key="parchment"/>
+                                       </support>
+                                       <extent>
+                                           <measure unit="leaf"></measure>                                            
```

Comment:
no number of leaves?

Reply:
No. I provided the incipit nevertheless because the matter of Dersana mahyawi is very confusing.